spec/acceptance/resource_cmd_spec.rb

```diff
@@ -64,6 +64,23 @@
     end
   end
 
+  if default['platform'] !~ %r{centos-5}
```

Comment:
What do you think @david22swan?

Reply:
@mateusz-gozdek-sociomantic Looks good but RedHat 5 needs to be excluded as well. Also I would prefer it if you where to use the syntax that I gave you, just to maintain consistency with the rest of the module.

Reply:
Right, I forgot about RedHat 5. Hmm, speaking of consistency, it seems that since we don't want to disable `describe`, but only one specific `context`, `if` should be used instead of `unless`. Example: https://github.com/puppetlabs/puppetlabs-firewall/blob/master/spec/acceptance/purge_spec.rb#L128-L129

It seems for me, that using `el-5` should be the right choice in this case.

BTW, feel free to push any code to this PR or suggest specific changes if this can make things faster :)

Reply:
Hm, I thought matching for `el-5` will disable all follwoing platforms from testing:
```
spec/acceptance/nodesets/new/pe/centos-5-64mda.yml:    platform: el-5-x86_64
spec/acceptance/nodesets/new/pe/oracle-5-64mda.yml:    platform: el-5-x86_64
spec/acceptance/nodesets/new/pe/redhat-5-64mda.yml:    platform: el-5-x86_64
spec/acceptance/nodesets/new/pe/scientific-5-64mda.yml:    platform: el-5-x86_64
```

Anyway, that's for posting the snippet, I'll give it a try locally and then push if it works. 

>Sorry for the wait, had some other stuff to handle, also if I push to your fork then I can't merge your pr, it's why I have been avoiding doing it.

No worries about the timing, but out of curiosity, why couldn't you merge it then?

Reply:
It's a company rule, except in certain situations, i..e. on new modules you are making or simple doc fixes, you are not allowed to merge your own PRs or any PR you have contributed to. It's to ensure that no error's are missed.